packages/longhorn/generated-changes/patch/Chart.yaml.patch

```diff
@@ -1,9 +1,9 @@
 --- charts-original/Chart.yaml
 +++ charts/Chart.yaml
 @@ -26,3 +26,12 @@
- - name: Sheng Yang
-   email: sheng@yasker.org
- icon: https://raw.githubusercontent.com/cncf/artwork/master/projects/longhorn/icon/color/longhorn-icon-color.svg?sanitize=true
+ - name: Longhorn maintainers
+   email: maintainers@longhorn.io
+ icon: https://raw.githubusercontent.com/cncf/artwork/master/projects/longhorn/icon/color/longhorn-icon-color.png
 +annotations:
```

Comment:
We want to support Longhorn installation in Rancher windows cluster. So we may want to remove this annotation: `catalog.cattle.io/os: linux`

Reply:
Yes if it supports both windows and linux you would remove that annotation

Reply:
Let's wait for @cclhsu update before merging.

Reply:
Just to be clear. Longhorn doesn't support Windows. The support we're adding is to allow Longhorn to be installed in the cluster with mixed Linux and WIndows node.

So we should NOT remove this annotation. @cbron @PhanLe1010 please confirm the understanding is right here.

Reply:
Windows here would imply that it works in some way with the windows nodes. Both logging and monitoring interact with the windows nodes. If longhorn is not interacting with the windows nodes and just tolerates them then keep the Linux annotation. 

Also wait to merge until I or another committer approve, there are a lot of requirements around charts in rancher releases and I need to ensure they are met.

Reply:
@yasker @cbron 

Yes, Longhorn doesn't run on Windows nodes. We want to allow the user to install Longhorn on Linux nodes in a cluster that have both Linux and Windows nodes.

If we don't the remove `catalog.cattle.io/os: linux` annotation, Longhorn chart will not be shown in the new Rancher UI when the cluster have both Windows nodes and Linux nodes. So, user cannot install Longhorn even if their cluster have some Linux nodes.

Before removing the `catalog.cattle.io/os: linux` annotation, new Rancher UI shows:
<img width="1066" alt="Screen Shot 2021-04-27 at 8 25 05 PM" src="https://user-images.githubusercontent.com/22139961/116342165-01fa1900-a797-11eb-800f-26e673f3ac97.png">

After removing the `catalog.cattle.io/os: linux` annotation, new Rancher UI shows:
<img width="1065" alt="Screen Shot 2021-04-27 at 8 24 45 PM" src="https://user-images.githubusercontent.com/22139961/116342180-09b9bd80-a797-11eb-9f7b-9473f0af9a4f.png">

Reply:
> My understanding is that we still show Longhorn chart in Windows cluster. When users install Longhorn, they need to specify toleration and node selector so that Longhorn components is only deployed on the Linux nodes as described in the documentation at https://longhorn.io/docs/1.1.1/advanced-resources/deploy/rancher_windows_cluster/

This is what I expect. @yasker any further concerns?

Reply:
BTW, this frontend implementation is from the below snippets, so obviously if you don't remove this annotation, there is no chance to install longhorn from the hybrid (windows+linux) cluster. (note: the OS annotation is exclusive or empty to allow in the hybrid cluster mode)

https://github.com/rancher/dashboard/blob/176ffcedfd57af42dab27b5c9c7723e8eab1590f/store/catalog.js#L527-L533 and https://github.com/rancher/dashboard/blob/176ffcedfd57af42dab27b5c9c7723e8eab1590f/store/catalog.js#L477-L491

Reply:
@khushboo-rancher @meldafrawi I have updated the test branch  to remove the `linux` annotation.
```
repo: https://github.com/cclhsu/charts
branch: dev-v2.5
```
Please help validate.  Thanks!

Reply:
@PhanLe1010 I see. It makes sense. Do we have documents on how users can deploy Longhorn in a mixed environment?

@cbron Can you confirm this is expected behavior? So adding the label means it's going to be installed in Linux only cluster. And if we want to support an environment mixed with Windows and Linux, then we need to remove the annotation and ask users to add node selectors to the chart.

Reply:
@cbron Is there a middle way to have option to install Longhorn on a Linux or mixed cluster and doesn't show this for a cluster having only windows nodes?

Reply:
>  Do we have documents on how users can deploy Longhorn in a mixed environment?
Yeah, we do have the document at https://longhorn.io/docs/1.1.1/advanced-resources/deploy/rancher_windows_cluster/

Reply:
@khushboo-rancher There is no such thing as a cluster with only Windows nodes; you need Linux for the system services.  That's why the filtering is just by "is the cluster Windows or not".

Reply:
This is technically ok, as long as users don't get confused. They might see it show up in a windows cluster and expect it to work on windows nodes. There will be no way for them to know which way it works.

Reply:
TLDR: After talking with @cbron , we agree we should keep the annotation there to prevent Longhorn from showing up for the Windows/Linux mixed cluster.

There were some misunderstandings regarding the original request filed by @cbron at https://github.com/rancher/rancher/issues/29220 . The requirement is: Any Rancher featured chart shouldn't be immediately broken when installing in a Windows/Linux mixed cluster, e.g. trying to run a Linux container on top of Windows nodes.

However, Rancher also doesn't recommend running any featured charts for a system it's not designed for. In this case, obviously, Longhorn isn't designed for Windows, so we shouldn't remove the annotation.

But if Rancher doesn't recommend that, why is it required? It's because users can still deploy Helm chart without Rancher UI, e.g. via API/Terraform/Helm. Those ways are not blocked in any sense. There are users who want to do that. But since it's not a common case to deploy Linux specific workload in a Windows/Linux mixed cluster, as the current policy, we don't encourage the user to do it and we want to make sure user knows what they're doing. If they choose to deploy the chart via API/Terraform/Helm, we reasonably believe that they know what they're doing.

If a chart just shows up in user's Windows/Linux mixed cluster, the users might just click and deploy it, then error out - since they need to set related `NodeSelector` and `TaintToleration` manually. It's not a good experience anyway. So it's easier to just not showing it unless the users explicitly deploy it via other means.

Longhorn is not designed for Windows. So we think it's better if we keep the annotation `catalog.cattle.io/os: linux`.

@PhanLe1010 @cclhsu Let me know what do you think.

Reply:
Thanks @yasker ! I agree with the conclusion